### PR TITLE
copr: fix unstable test

### PR DESCRIPTION
**Pull request by zhongzc**
Signed-off-by: Zhenchi <zhongzc_arch@outlook.com>

### What problem does this PR solve?

The deadline test can fail unexcepted. Adjust it.

### Check List 

Tests 

- Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

File: src/coprocessor/endpoint.rs [hunk 1]
```diff
@@ -735,12 +735,12 @@ mod tests {
     impl RequestHandler for UnaryFixture {
         async fn handle_request(&mut self) -> Result<coppb::Response> {
             if self.yieldable {
-                // We split the task into small executions of 1 second.
-                for _ in 0..self.handle_duration_millis / 1_000 {
-                    thread::sleep(Duration::from_millis(1_000));
+                // We split the task into small executions of 1 millisecond.
```
Comment (sticnarf) [inline src/coprocessor/endpoint.rs:738]: emm, in the code it seems to be split into executions of 100ms?

Comment (zhongzc) [reply src/coprocessor/endpoint.rs:738]: yes...

Comment (zhongzc) [reply src/coprocessor/endpoint.rs]: Updated

**Review by sticnarf — APPROVED** [review]

**Review by andylokandy — APPROVED** [review]

**Pull request merged**